discord/guild.py

```diff
@@ -2064,3 +2064,23 @@ async def query_members(self, query=None, *, limit=5, user_ids=None, cache=True)
 
         limit = limit or 5
         return await self._state.query_members(self, query=query, limit=limit, user_ids=user_ids, cache=cache)
+
+    async def change_voice_state(self, channel, *, self_mute=False, self_deaf=False):
```

Comment:
```suggestion
    async def change_voice_state(self, *, channel, self_mute=False, self_deaf=False):
```

Reply:
Since `channel` is required parameter, I thought that it might a little bit bothersome to have to use keyword arguments here; unless a sentinel is added that just uses the channel from current voice state?

Reply:
It's because `change_voice_state(None)` is a terrible read.